doc/Extensions/Applications.md

```diff
@@ -469,3 +470,20 @@ if [ "$1" = "config" ]; then
 fi
 echo -n "foobar.value " $(date +%s) #Populate a value, here unix-timestamp
 ```
+
+
+#### PHP-FPM
+
+##### SNMP Extend
+
+1. Copy the shell script, phpfpm-sp, to the desired host (the host must be added to LibreNMS devices)
```

Comment:
Please add the command to download the file like: https://github.com/VVelox/librenms/blob/3a10929a8f62652cc1fc846f38c1da98395bbfd2/doc/Extensions/Applications.md#proxmox